tests/page.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var webdriver = require('selenium-webdriver');
-
+var idSelectors = true;
```

Comment:
It would be more logical if this defaults to `false`. The usage of IDs is temporary, all examples will need to be switched over one day.